### ISSUE TITLE
Cannot query compilation status

### ISSUE THREAD

**Issue by kelokchan** [shr3jn/fastlane-plugin-huawei_appgallery_connect#38]
I'm getting this error while using the fastlane script

```
[!] Cannot query compilation status (status code: 400, body: {"message":"Parameter is not valid for operation [AppGalleryConnectPublishService.publish-services.getPackageCompileStatus]. Parameter is [pkgIds]. Processor is [query]."})
```

My script looks something like this
```
huawei_appgallery_connect(
      client_id: options[:client_id],
      client_secret: options[:client_secret],
      app_id: 123,
      apk_path: /path
      is_aab: true,
      delay_before_submit_for_review: 120,
      changelog_path: "fastlane/metadata/android/en-US/changelogs/default.txt"
    )
```

The `aab` was successfully uploaded but I couldn't query the compilation status.

**Comment by shr3jn** [comment]
please confirm if you're passing correct parameters.

**Comment by kelokchan** [comment]
> please confirm if you're passing correct parameters.

I think so yeah? The same script used to work completely fine. I also have the mandatory parameters correctly set

**Comment by DaniyarAmangeldy** [comment]
I have same error

**Comment by shr3jn** [comment]
It seems new parameter is added on the Huawei API for querying the compilation status of AAB. I've published new version 1.0.21 which now allows to define package ids as:

`package_ids: "package id here"`

but it's not clear what the package id is. I tried setting it as package name. It did not work on my test. Maybe, you guys could give it a go?

**Comment by shr3jn** [comment]
https://developer.huawei.com/consumer/en/doc/development/AppGallery-connect-References/agcapi-query-aabfile-0000001111685206

<img width="943" alt="Screen Shot 2022-05-21 at 19 49 16" src="https://user-images.githubusercontent.com/4709201/169655187-6eee6e2a-5419-4560-b285-e2978e1c550d.png">

**Comment by shr3jn** [comment]
fixed on version 1.0.23. No changes required on config.
